dandiapi/api/services/asset/paths/__init__.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
```

Comment:
I wouldn't put any of the code in this PR underneath `services`. With the exception of `search_asset_paths`, none of these are really user facing business logic. Given we're still not putting read operations into the services module either, I think all of these should be put in something like `dandiapi.api.asset_paths`.

Reply:
https://github.com/dandi/dandi-archive/pull/1312/commits/a2c9444961e5ef8aa3ba4cb89120beafd0544876